main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -3408,6 +3411,86 @@ public static void saveLists(final Collection<Geocache> caches, final Set<Intege
         }
     }
 
+    public static Set<String> getStoredLocationsInList(final int listId) {
+
```

Comment:
this could get all locations in a stored list I guess. may be dropped if wholly unnecessary.